phys2bids/tests/test_utils.py

```diff
@@ -66,7 +67,7 @@ def test_move_file(tmpdir):
         pass
```

Comment:
I can't comment at the right line, but I would equally remove this chunk of code (lines 62-70)